stock_warehouse_orderpoint_stock_info_unreserved/tests/test_stock_warehouse_orderpoint.py

```diff
@@ -1,61 +1,63 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Eficent Business and IT Consulting Services, S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests.common import TransactionCase
+from odoo.tests.common import SavepointCase
 
 
-class TestStockWarehouseOrderpoint(TransactionCase):
-
-    def setUp(self):
-        super(TestStockWarehouseOrderpoint, self).setUp()
+class TestStockWarehouseOrderpoint(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Just a candid question: what's the point of renaming ``self`` to ``cls``?

Edit: OK, it's a pep8 recommendation, see https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments
> Always use self for the first argument to instance methods.
> Always use cls for the first argument to class methods.